### ISSUE TITLE
Model not saving in Google Drive after training

### ISSUE THREAD

**Issue by katreparitosh** [googlecolab/colabtools#1348]
**Current behavior:**

I trained a BERT model on Colab. The model works fine as expected. But the **model is not saving on my Google Drive.** I re-checked the path, commands multiple times, I also tried different locations, model names, but its not saving. I also waited for hours just to be sure. 

While other methods like converting a dataframe to csv and saving it on drive is working fine. The csv is getting saved. 

**Expected behavior:**

Earlier, whenever I practiced this procedure of saving my model on the drive, it worked smoothly. But since the last 2 projects, it is not working.  

I am using Google Chrome.

Kindly help.

**Comment by ramani41** [comment]
06/30/20; I have a problem with saving trained model.

I want to save the trained model in the protobuf format. I understand that this is the default mode in TF2.2 which I am using in my Colab notebook. When I save it only saves in HDF5 format. Here is the cell in which model is saved:
***********
print(time.ctime())
print('VGG16 model with layer training false')
batch_size =50
epochs = 5
model.fit(X_train, y_train, batch_size=batch_size,epochs=epochs,verbose=2,validation_data=(X_val, y_val), shuffle=True)
# Save the entire model as a SavedModel.
print('tf version ',tf.version.VERSION)
os.chdir('/content')
!mkdir -p pb-saved_model
model.save('pb-saved_model/my_pbmodel')
**************
However, when I follow https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/keras/save_and_load.ipynb#scrollTo=sI1YvCDFzpl3. and save the trained model it correctly saves in protobuf format. This notebook also runs TF2.2. Here is the cell that saves the model:
*******
# Create and train a new model instance.
model = create_model()
model.fit(train_images, train_labels, epochs=5)
print('tf version ',tf.version.VERSION)
# Save the entire model as a SavedModel.
!mkdir -p pb-saved_model
model.save('pb-saved_model/my_pbmodel') 
******

I am stumped; need help.
